lemur/plugins/lemur_gcp/plugin.py

```diff
@@ -0,0 +1,107 @@
+from flask import current_app
+from google.cloud.compute_v1.services import ssl_certificates
+from google.oauth2 import service_account
+import google.auth
+import hvac
+import os
+
+from lemur.plugins.bases import DestinationPlugin
+from lemur.plugins import lemur_gcp as gcp
+
+
+class GCPDestinationPlugin(DestinationPlugin):
+    title = "GCP"
+    slug = "gcp-destination"
+    version = gcp.VERSION
+    description = "Allow the uploading of certificates to GCP"
+    author = "Mitch Cail"
+    author_url = "https://github.com/Datadog/lemur"
+
+    options = [
+        {
+            "name": "accountId",
```

Comment:
Nit: Call this `projectID` to match GCP nomenclature?

Reply:
Noting here the frontend parsing logic makes it render as "Project I D".